lib/apiservers/engine/backends/volume.go

```diff
@@ -234,7 +242,7 @@ func translateInputsToPortlayerRequestModel(name, driverName string, opts, label
 	model.Metadata = make(map[string]string)
 	model.Metadata[dockerMetadataModelKey] = metadata
 	if err := validateDriverArgs(opts, model); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Bad Driver Arg: %s", err)
```

Comment:
e.g. `Bad Driver Arg: Invalid Capacity: HelloWorld` for a `--opt Capacity=HelloWorld`